docker/app/Dockerfile

```diff
@@ -1,26 +1,84 @@
-FROM sillsdev/web-languageforge:app-base-staging
+ARG ENVIRONMENT=production
+
+# APP-BASE
+FROM php:7.3.28-apache AS base-app
+
+# install apt packages
+# p7zip-full - used by LF application for unzipping lexicon uploads
+# unzip - used by LF application for unzipping lexicon uploads
+# git - TODO: why do we need git?
+# gnupg2 - necessary for LFMerge package installation via SIL sources (will be uninstalled in production)
+# curl - used by LF application
+RUN apt-get update && apt-get -y install p7zip-full unzip git gnupg2 curl
+
+# see https://github.com/mlocati/docker-php-extension-installer
+# PHP extensions required by the LF application
+COPY --from=mlocati/php-extension-installer /usr/bin/install-php-extensions /usr/local/bin/
+RUN install-php-extensions gd mongodb intl
+
+# install and configure LFMerge
+RUN curl -L http://linux.lsdev.sil.org/downloads/sil-testing.gpg | apt-key add - \
+&& echo "deb http://linux.lsdev.sil.org/ubuntu bionic main" > /etc/apt/sources.list.d/linux-lsdev-sil-org.list \
+&& apt-get update \
+&& apt-get install --yes --no-install-recommends python lfmerge rsyslog logrotate cron iputils-ping
```

Comment:
there be an opportunity here to remove anything not needed anymore, e.g., apt-get cache or anything like that.  If something exists like that, it should of course be done as part of a compound statement here.

Reply:
@rmunn #1034 creates a bash script which runs indefinitely in a loop - a simple man's cronjob.  This was to avoid having the cron dependency, however I see in this line that we do install cron.  I'm wondering if A) we can remove the cron dependency or B) we should just switch over to using cron since we've already got it.  Would prefer to remove cron if that was the only thing it's needed for - looking for your insight into this one.  Thanks.